Doctrine/Subscriber/AbstractEntitySubscriber.php

```diff
@@ -250,7 +250,7 @@ protected function flushRemovals(): void
                     // If the entity isn't able to be sent to Salesforce for any reason,
                     // a RuntimeException is thrown. We don't want that stopping our fun.
                     $this->logger->error($e->getMessage());
-                    $this->logger->debug($e->getTraceAsString());
+                    $this->logger->debug('AbstractEntityCompiler-001 - '.$e->getTraceAsString());
```

Comment:
Can you change this to "Exception occurred while flushing removals " or "Exception occured in flushRemovals" or something along those lines?